web/concrete/single_pages/profile/avatar.php

```diff
@@ -64,30 +61,31 @@ function checkCoords(){
                 $('#real_w').val(pic_real_width);
                 $('#real_h').val(pic_real_height);
                 function updatePreview(c){
-                  if (parseInt(c.w) > 0){
-                    $.each($preview, function($index, $avatar){
-                      var rx = $($avatar).parent().width() / c.w;
-                      var ry = $($avatar).parent().height() / c.h;
-                      $($avatar).css("width",Math.round(rx * boundx) + 'px')
-                        .css("height",Math.round(ry * boundy) + 'px')
-                        .css("marginLeft",-Math.round(rx * c.x) + 'px')
-                        .css("marginTop",-Math.round(ry * c.y) + 'px');
-                    });
-                  }
+                  //if (parseInt(c.w) > 0){
```

Comment:
Are you uncertain about this code? Is there a chance that we might need it at some point? If not I would remove it. Having commented code without a comment about the reason for the comments will only confuse code-readers.

Reply:
this is in removal of the placeholder and preview.   For implication purposes I left it there for the moment and could be removed to my knowledge.  The preview only worked when the user did not have an avatar set, and it didn't want to play along with AVATAR_NONE or with responsive themes (Mainly tested with Marketplace Bootstrap theme).  The two lines above are the two I can say I'm a bit unsure of.  Was getting null values and throwing mysql error on first avatar upload.  Omitting these two lines resolves the issue ?!  As long as things look good from your perspective I'll get it removes so we can merge...

Reply:
I just started playing with 5.8 recently.  I like this implementation MUCH better than that in 5.8!

Reply:
Well, I haven't debugged it and I don't know much about how it works, I just see commented code I'm not understanding ;-) I always like to understand why code is causing a problem, but it has to wait, got some pressing tasks